internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -162,7 +179,7 @@ func (q amsQuotaService) getAvailableBillingModelFromKafkaInstanceType(externalI
 	return billingModel, nil
 }
 
-func (q amsQuotaService) getBillingModel(kafka *dbapi.KafkaRequest, instanceType types.KafkaInstanceType) (string, error) {
+func (q amsQuotaService) GetBillingModel(kafka *dbapi.KafkaRequest, instanceType types.KafkaInstanceType) (string, error) {
```

Comment:
Is this change needed?

Reply:
Thanks @pb82 since it is a unit test within the package it should be okay to still keep it a private and test it, what do you think?

Reply:
you're absolutely correct @machi1990 , i've updated this now.